oak_derive/tests/visit.rs

```diff
@@ -131,9 +131,13 @@ mod enums {
 fn assert_visit<T: HandleVisit>(mut t: T, count: usize) {
     let mut counter = 0;
 
-    t.visit(|_| {
-        counter += 1;
-    });
+    // TODO(#1599): Re-enable when the underlying issue is fixed.
+    #[allow(unused_must_use)]
```

Comment:
Can you use `let _ = ` here instead? See also: https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute

Reply:
Done.